modules/fts/pages/fts-geospatial-queries.adoc

```diff
@@ -6,21 +6,32 @@ Geospatial queries return documents that contain location-data.
 [#introduction-to-geospatial-querying]
 == Introduction to Geospatial Querying
 
-A _geospatial query_ specifies a location, and returns each document that contains a proximity-match.
-A location is represented by means of _longitude_-_latitude_ coordinate pairs.
+A _geospatial query_ specifies an area, and returns each document that contains a reference to a location within the area.
+Areas and locations are represented by means of _latitude_-_longitude_ coordinate pairs.
 
-This allows an application, based on the user's input of a particular coordinate, address, or property name, to derive a list of _points of interest_ that lie within a specified distance of the user-referenced location; and display these for the user's benefit.
+This allows an application, based on the user's input of a particular coordinate, address, or property name, to derive a list of _points of interest_ that lie within an area that surrounds the user-referenced location; and display these for the user's benefit.
 
-The location-data provided by a geospatial query can be either of the following:
+A geospatial query can specify an area by means of:
 
-* A location, specified as a longitude-latitude coordinate pair; and a distance, in miles.
+* A location, expressed as a latitude-longitude coordinate pair; and a distance, in miles.
```

Comment:
Formats of distance units supported:
"5in" "5inch" "7yd" "7yards" "9ft" "9feet" "11km" "11kilometers" "3nm" "3nauticalmiles" "13mm" "13millimeters" "15cm" "15centimeters" "17mi" "17miles" "19m" "19meters"

If the unit cannot be determined, the entire string is parsed and the unit of meters is assumed.